crates/chia-puzzles/src/proof.rs

```diff
@@ -22,6 +22,15 @@ pub struct LineageProof {
 #[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
 #[clvm(list)]
 pub struct EveProof {
+    pub parent_parent_coin_id: Bytes32,
```

Comment:
this is usually called `parent_coin_info`, not a great name, but it introduces an inconsistency to use a different name

Reply:
oh, I misread. this is the parent's parent